sim_city.py

```diff
@@ -0,0 +1,116 @@
+class City:
+    """
+    Represents a city in Ackland, should help calculate the tax of each city
+    """
+
+    def __init__(self):
+        self.neighborhoods = {}
+        self.base_tax = 1000
+
+    def add_new_neighborhood(self, neighborhood_name: str):
+        """
+        Creates a new neighborhood in the city
+        :param neighborhood_name: the name of the neighborhood
+        """
+        self.neighborhoods[neighborhood_name] = Neighborhood()
```

Comment:
A neighborhood should probably be aware of its own name as well (for printing purposes).